x-pack/plugins/apm/server/routes/mobile/route.ts

```diff
@@ -66,6 +74,61 @@ const mobileFiltersRoute = createApmServerRoute({
   },
 });
 
+const mobileChartsRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/mobile-services/{serviceName}/most_used_charts',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([
+      kueryRt,
+      rangeRt,
+      environmentRt,
+      t.partial({
+        transactionType: t.string,
+      }),
+    ]),
+  }),
+  options: { tags: ['access:apm'] },
+  handler: async (
+    resources
+  ): Promise<{
+    mostUsedCharts: Array<{
+      key:
+        | MobileMostUsedChartResponse[number]['key']
```

Comment:
[nit] Couldn't we just merge these two types in just one?

Reply:
Yes, this should be fixed now with the ENUM implementation.
Not sure why its not showing as Outdated

Reply:
Maybe I was reviewing while you were updating the PR